src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
```

Comment:
Hm; Well really the question is - what is the intent of the tests?

Is it purely to assert that the className is set in some way? If so, you could simply use `not.toBeUndefined();` or `toContain('foo');`